ledger/acctupdates.go

```diff
@@ -241,7 +241,7 @@ func (e *RoundOffsetError) Error() string {
 }
 
 // StaleDatabaseRoundError is generated when we detect that the database round is behind the accountUpdates in-memory dbRound. This
-// should never happen, since we update the database first, and only upon a successfull update we update the in-memory dbRound.
+// should never happen, since we update the database first, and only upon a successful update we update the in-memory dbRound.
```

Comment:
Similar typos:
catchup/catchpointService.go:410:		// give a rank to the download, as the download was **successfull**.
network/wsPeer.go:439:				// do nothing. writing was **successfull**.
scripts/buildhost/start_ec2_instance.sh:13:# Upon **successfull** execution, the following files would be created:
test/framework/fixtures/libgoalFixture.go:105:// nodeExitWithError is a callback from the network indicating that the node exit with an error after a **successfull** startup.

Reply:
fixed.